patches/net/minecraft/client/gui/font/glyphs/BakedGlyph.java.patch

```diff
@@ -0,0 +1,15 @@
+--- a/net/minecraft/client/gui/font/glyphs/BakedGlyph.java
++++ b/net/minecraft/client/gui/font/glyphs/BakedGlyph.java
+@@ -75,6 +_,12 @@
+         p_95223_.addVertex(p_254370_, p_95221_.x0, p_95221_.y1, p_95221_.depth).setColor(p_95221_.color).setUv(this.u1, this.v0).setLight(p_95224_);
+     }
+ 
++    public RenderType renderType(Font.DisplayMode p_181388_, boolean blur) {
```

Comment:
[Similarly](https://github.com/neoforged/NeoForge/pull/1604/files#r1833690172) for this one